linode/instance/tmpl/templates/instance_with_reserved_ip.gotf

```diff
@@ -0,0 +1,15 @@
+{{ define "instance_with_reserved_ip" }}
+
+resource "linode_instance" "foobar" {
+    label  = "{{ .Label }}"
+    type   = "g6-nanode-1"
+    region = "{{ .Region }}"
+    image  = "{{ .Image }}"
+
+    root_pass = "{{ .RootPass }}"
+    authorized_keys = ["{{ .PubKey }}"]
+
+    ipv4 = [{{ range $index, $ip := .IPv4 }}{{ if $index }}, {{ end }}"{{ $ip }}"{{ end }}]
```

Comment:
Can you add the e2e test firewall to the instance for compliance:

```
    firewall_id = linode_firewall.e2e_test_firewall.id

```

Reply:
I have now added the firewall to the the linode_instance for compliance reasons. Thanks!